tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationView.java

```diff
@@ -256,10 +263,28 @@ private TableViewerColumn createTableViewerColumn(
         column.setWidth( bound );
         column.setResizable( true );
         column.setMoveable( true );
+        column.addSelectionListener(getSelectionAdapter(column, colNumber));
```

Comment:
This doesn't follow the format/style,  see the method calls above and then this one.